app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php

```diff
@@ -33,7 +33,32 @@
 --}}
 
 <div class="flex gap-2">
+    @if ($getRecord()->status)
+        <x-filament::badge>
+            {{ $getRecord()->status->name }}
```

Comment:
Is status an option field right now? If so, we will probably want to keep the `$getRecord()?->status?->name ?? ''` pattern you had before to prevent an error being thrown when visiting an article that does not have a status set.

Reply:
yes, status is optional field. so i have added like you suggest. but we cannot remove this @if ($getRecord()->status) condition because if we remove that condition then it's print blank badge on screen.